src/main/java/io/icker/factions/database/Database.java

```diff
@@ -24,6 +24,7 @@ color VARCHAR(255),
                 CREATE TABLE IF NOT EXISTS Member (
                     uuid UUID PRIMARY KEY,
                     faction VARCHAR(255),
+                    rank VARCHAR(255),
```

Comment:
Does this have to be a VARCHAR? I'd say ENUM would be more suitable.

Reply:
Possibly. I tried to use ENUM, but I just couldn't get it to work. Maybe you could. Also, sorry for the late response, I don't check github too often.